addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -2196,6 +2196,28 @@ export class PosStore extends Reactive {
     get showSaveOrderButton() {
         return this.isOpenOrderShareable();
     }
+
+    async isSessionDeleted() {
+        const session = await this.data.read("pos.session", [this.session.id]);
+        return session[0] === undefined;
+    }
+
+    async handleSessionDeleted() {
```

Comment:
Maybe you can also use this method in `closingSessionNotification` and replace the current dialog.

Reply:
@davidmonnom The messages in `closingSessionNotification` are different and there are two cases. I think it's not needed to use the same method.

Reply:
@davidmonnom It’s also possible to have a missing error for another record, so it’s safe to check for session deletion.